pkg/morph/deploy/contracts.go

```diff
@@ -126,11 +135,54 @@ func syncNeoFSContract(ctx context.Context, prm syncNeoFSContractPrm) (util.Uint
 		Sender() util.Uint160
 	}
 	var managementContract *management.Contract
-	if prm.committeeDeployRequired {
-		deployCommitteeActor, err := newCommitteeNotaryActorWithCustomCommitteeSigner(prm.blockchain, prm.localAcc, prm.committee, func(s *transaction.Signer) {
-			s.Scopes = transaction.CustomContracts
-			s.AllowedContracts = append(prm.extraCommitteeDeployAllowedContracts, prm.nnsContract)
-		})
+	if prm.deployWitness > 0 {
+		if prm.deployWitness != witnessAlphabet && prm.deployWitness != witnessAlphabetAndCommittee {
+			panic(fmt.Sprintf("unexpected deploy witness mode value %v", prm.deployWitness))
+		}
+
+		committeeDefaultMultiSigAcc := wallet.NewAccountFromPrivateKey(prm.localAcc.PrivateKey())
```

Comment:
Technically, it's not a committee address. It's a validator multisig (when you take validators only, of course, so `ValidatorsCount` can be applied, although NeoFS has it set to the size of the committee). Your initial NEO and GAS reside on this account, for example.

Reply:
ok i agree, competely do not like inconsistency of terms

btw what u think of Alphabet? it's mentioned in many places like https://pkg.go.dev/github.com/nspcc-dev/neofs-contract@v0.19.1/netmap#SubscribeForNewEpoch

Reply:
I doubt NeoFS will survive in `len(committee) != ValidatorsCount` scenario now, btw. But it's a bit different story.

Reply:
left "validators" for now as more Neo-known term